app/src/lib/stores/app-store.ts

```diff
@@ -1826,6 +1826,36 @@ export class AppStore extends TypedBaseStore<IAppState> {
         this.statsStore.recordCoAuthoredCommit()
       }
 
+      const selectedState = this.getSelectedState()
+      if (
+        selectedState !== null &&
+        selectedState.type === SelectionType.Repository &&
+        selectedState.repository.gitHubRepository !== null
+      ) {
+        const { commitAuthor } = state
+        if (commitAuthor !== null) {
+          const account = getAccountForRepository(
+            this.accounts,
+            selectedState.repository
+          )
+          if (account !== null) {
+            const email = account.emails.find(
+              email =>
+                email.email.toLowerCase() === commitAuthor.email.toLowerCase()
+            )
+            if (email == null) {
+              this.statsStore.recordUnattributedCommit()
+            } else if (
```

Comment:
Since we've introduced these two new metrics (i.e. `gheCommits` and `dotcomCommits`) we should probably increment them regardless of whether the commit will go unattributed or not, right? So we should move those two out of the if-statement checking for a `commitAuthor` and increment one of them as long as there's an account available for the repository.